src/main/java/me/zeroeightsix/fiber/tree/ConfigValue.java

```diff
@@ -20,12 +26,23 @@
     private final BiConsumer<T, T> consumer;
     @Nonnull
     private final List<Constraint<? super T>> constraintList;
-
-    private final Predicate<T> restriction;
-
     @Nonnull
     private final Class<T> type;
+    private final Predicate<T> restriction;
 
+    /**
+     * Creates a {@code ConfigValue}.
+     *
+     * @param name the name for this item
+     * @param comment the comment for this item
+     * @param value the current value
+     * @param defaultValue the default value for this item
+     *                     <br> Preferably, if this item's value has constraints, the default value must satisfy those constraints.
```

Comment:
The words preferably and must appear to be contraditory

Reply:
I'll change *must* to *should*.